### PR TITLE
add: pure numpy

### PR DESCRIPTION
**Pull request by kkweon**
## Summary
* Create a pure numpy version of DeepLearningZeroToAll
* Currently, two are implemented:
  * logistics regression
  * softmax classification
* In the future, 
  * Adam & RMSProp Optimizer
  * Dropout & Batch Normalization
  * CNN, RNN

### CODE REVIEW

**Comment by hunkim** [comment]
This is just lovely! Super!!
